akd/tools/source_validator.py

```diff
@@ -21,16 +23,15 @@
 from akd.tools._base import BaseTool, BaseToolConfig
 from akd.utils import get_akd_root
 
-if TYPE_CHECKING:
-    pass
-
 
 class SourceInfo(BaseModel):
     """Schema for source information from CrossRef API."""
 
-    title: str = Field(..., description="Source title")
+    title: str = Field(..., description="Source title (journal/proceedings)")
     publisher: Optional[str] = Field(None, description="Publisher name")
-    issn: Optional[List[str]] = Field(None, description="List of ISSNs")
+    issn: List[str] = Field(
+        default_factory=list, description="List of normalized ISSNs"
+    )
```

Comment:
I see issn changed from list[str] to optional[list[str]]. Not sure what are the consequences. Are we guaranteeing some issn?